src/libstd/time/duration.rs

```diff
@@ -207,9 +259,11 @@ impl Duration {
         }
     }
 
-    /// Checked duration division. Computes `self / other`, returning `None`
+    /// Checked `Duration` division. Computes `self / other`, returning [`None`]
     /// if `other == 0` or the operation results in underflow or overflow.
```

Comment:
Drive-by fix: division cannot underflow or overflow in `other != Duraton::from_secs(0)` as having a negative or floaty duration is not possible.